src/graphql/fragments/groupQueryFragment.js

```diff
@@ -159,6 +159,7 @@ export default () => {
       items {
         id
         title
+        type
```

Comment:
Just curious why this came-up too?

Reply:
same issue, not having the type was breaking the page completely. we could just load all the post stuff here, but i was trying to reduce the load time of the explore page where this is used by only loading what is needed for the post cards there.